lib/services/sf.js

```diff
@@ -578,17 +579,27 @@ function StateAbstract(options) {
    *
    * @param {Object} requestOptions
    * @param {Object} httpClient
-   *
+   * @param {Object} auth
+   * @param {boolean} includeGuid
    * @returns {Object} the http request object.
    */
-  function sendHttpRequest(requestOptions, httpClient, auth) {
+  function sendHttpRequest(requestOptions, httpClient, auth, includeGuid = true) {
```

Comment:
In which situation it can be `false`? In Go driver we don't add guid when sending inband telemetry, but I can't see that you invoked this function with `false`. It would be good to verify if we should add this param in telemetry requests (and fix it in go driver).